webapp/views/App/Header/Header.js

```diff
@@ -62,7 +62,9 @@ const Header = () => {
                   dispatch(UserActions.updateUserPrefs({ user: userUpdated }))
                 }}
               />
-              {Survey.isDraft(surveyInfo) && <ButtonPublishSurvey className="btn-secondary" />}
+              {Survey.isDraft(surveyInfo) && !Survey.isTemplate(surveyInfo) && (
+                <ButtonPublishSurvey className="btn-secondary" />
```

Comment:
add the check to ButtonPublishSurvey directly to avoid this check everytime you use the ButtonPublishSurvey component

Reply:
ok, good